app/assets/javascripts/application.js

```diff
@@ -12,5 +12,5 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require turbolinks
+// require turbolinks
```

Comment:
turbolinks preserves header and allows for faster internet speeds because header is not changed again and again and only body and title tags are changed.